# Initial state.
px: 0
py: 0
pz: 0
# Below init quaternion for `awr_6.bag`.
#qw: 0.999824
#qx: -0.000124
#qy: -0.015638
#qz: -0.010378
# Below init quaternion for `awr_7.bag`.
#qw: 0.999863
#qx: -0.001926
#qy: -0.014326
#qz: -0.008075
qw: 1
qx: 0
qy: 0
qz: 0
# Initial velocity.
vx: 0
vy: 0
vz: 0
# Initial IMU biases.
b_wx: 0
b_wy: 0
b_wz: 0
b_ax: 0
b_ay: 0
b_az: 0
# IMU noise parameters.
# NTNU params
noise_acc: 0.037264803241608045
noise_accbias: 0.009240244976898931
noise_gyr: 0.007374316585420674
noise_gyrbias: 0.0012879411924649577
#noise_acc: 0.0083
#noise_accbias: 0.00083
#noise_gyr: 0.0013
#noise_gyrbias: 0.00013
noise_qwv: 0.0
noise_qri: 0.0
noise_pri: 0.0
# Distance to feature.
noise_meas1: 0.04
# IMU.
noise_meas2: 0.01
# Velocity.
noise_meas3: 0.04
# Persistent feature noise. 0.4
noise_meas4: 0.04
# Below radar-IMU calibration for `awr_1 - 10` bags from 22.08.22.
q_riw: 0
q_rix: 0.953717
q_riy: 0
q_riz: -0.3007058
p_rix: 0.07771
p_riy: 0.02141
p_riz: -0.03631
# Not directly used parameters - needed for the autogenerated code.
noise_qwv: 0.0
noise_aux: 0.0
noise_scale: 0.0
# Noise parameters for persistent features. 0.1
noise_pf_x: 0.1
noise_pf_y: 0.1
noise_pf_z: 0.1
